docs/basics/troubleshooting.md

```diff
@@ -0,0 +1,56 @@
+# Troubleshooting first time setup
+
+## My site is showing "Internal Server Error"
+
+If you are seeing a blank screen with just the words "Internal Server Error" on it,
+it means that the django webworker process is not reporting as "down",
+but still failing to bootstrap fully.
+(If you are seeing a more elaborate 500 page, then that is an issue with a single request,
+but ususally does not indicate a more pervasive problem with the site's ability to receive and handle requests in general.)
+Often this is because it is unable
+using a backing service (such as PostgreSQL, CouchDB, or Redis).
```

Comment:
ES, Raik, pgbouncer

Reply:
"unable using" -> "unable to connect to"